docs/specification/current.md

```diff
@@ -268,9 +268,8 @@ No single FHIR resource represents a user, rather Practitioner and PractitionerR
 
 To reduce the implementation burden on CDS Clients that support CDS Services, this specification RECOMMENDS that prefetch queries only use a subset of the full functionality available in the FHIR specification. Valid prefetch templates SHOULD only make use of:
 
-* _instance_ level [read](https://www.hl7.org/fhir/http.html#read) interactions (for resources with known ids such as `Patient` and `Practitioner`)
+* _instance_ level [read](https://www.hl7.org/fhir/http.html#read) interactions (for resources with known ids such as `Patient`, `Practitioner`, or `Encounter`; e.g. `patient={{context.patientId}}`)
```

Comment:
"such as" implies an example, so I don't think we need to keep growing the list.  This line is also focused on instance-level retrieval (by id), so it wouldn't involve a search at the type level.